config/locales/client.en.yml

```diff
@@ -19,6 +22,19 @@ en:
       going_max: "Maximum guests"
       going_max_label: "{{goingMax}} Guests"
       error: "Event end should be same as or after event start"
+    event_label:
```

Comment:
Is there a reason why we're not using the existing day / hour / minute localisation in discourse/discourse? If we use that we also automatically get all of the translations, so there is a strong default to use that if the case is the same as it seems to be here.

https://github.com/discourse/discourse/blob/main/config/locales/server.en.yml#L943

Reply:
good catch!!

resolved and removed bespoke versions.